page-home.php

```diff
@@ -0,0 +1,52 @@
+<?php
+/*
+Template Name: Homepage
+*/
+get_header(); ?>
+
+<main id="primary" role="main" class="content-area">
+	<?php
+		global $post;
```

Comment:
To create tests we need to pass variables into the function to return something. If the variables are inside the function then there is nothing to pass into it.